pkg/controller/usersignup/masteruserrecord_test.go

```diff
@@ -104,56 +104,6 @@ func TestMigrateMurIfNecessary(t *testing.T) {
 		assert.True(t, changed)
 		assert.Equal(t, newExpectedMur(), *changedMur)
 	})
-
-	t.Run("when NSTemplateSet is missing templateRefs", func(t *testing.T) {
-		// given
-		nSTemplateTier := newNsTemplateTier("advanced", "654321b", "dev", "stage", "extra")
-		mur := newMasterUserRecord(nSTemplateTier, "johny", operatorNamespace, test.MemberClusterName, "123456789")
-		mur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces = []v1alpha1.NSTemplateSetNamespace{
-			{
-				Type:        "dev",
-				Revision:    "123abc1",
-				TemplateRef: "",
-			},
-			{
-				Type:        "stage",
-				Revision:    "123abc2",
-				TemplateRef: "",
-			},
-			{
-				Type:        "extra",
-				Revision:    "123abc3",
-				TemplateRef: "",
-			},
-		}
-		mur.Spec.UserAccounts[0].Spec.NSTemplateSet.ClusterResources = &v1alpha1.NSTemplateSetClusterResources{
-			Revision:    "654321b",
-			TemplateRef: "",
-		}
-
-		// when
-		changed, changedMur := migrateMurIfNecessary(mur, nSTemplateTier)
-
-		// then
-		assert.True(t, changed)
-		assert.Equal(t, newExpectedMur(), *changedMur)
-	})
-
-	t.Run("when one namespace is missing and one is extra, but rest is fine, then doesn't change", func(t *testing.T) {
-		// given
-		nSTemplateTier := newNsTemplateTier("advanced", "654321b", "dev", "stage", "extra")
-		mur := newMasterUserRecord(nSTemplateTier, "johny", operatorNamespace, test.MemberClusterName, "123456789")
-		mur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces[0].Type = "cicd"
-		mur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces[0].TemplateRef = "advanced-cicd-123abc1"
-		providedMur := mur.DeepCopy()
-
-		// when
-		changed, changedMur := migrateMurIfNecessary(mur, nSTemplateTier)
-
-		// then
-		assert.False(t, changed)
-		assert.Equal(t, *providedMur, *changedMur)
-	})
```

Comment:
May I know why did you remove this subtest?

Reply:
@MatousJobanek this was removed due to the removal of the migration of TemplateRef from `migrateMurIfNecessary`

Reply:
@MatousJobanek ohhh sorry, you're right, i've re-added it here: ef347313b48735e0e6baa1c6b8bc62e28491a3f0